tests/test_util.py

```diff
@@ -3,10 +3,22 @@
 # This file is part of gunicorn released under the MIT license.
 # See the NOTICE for more information.
 
+import pytest
+
 from gunicorn import util
 
 
-def test_parse_address():
+def test_parse_address(capsys):
     # Test unix socket addresses (PR #1623)
     assert util.parse_address('unix://var/run/test.sock') == 'var/run/test.sock'
     assert util.parse_address('unix:/var/run/test.sock') == '/var/run/test.sock'
+
+    assert util.parse_address('') == ('0.0.0.0', 8000)
```

Comment:
We could turn this test to use `pytest.mark.parametrize`: https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize